mkdocs.yml

```diff
@@ -73,6 +73,9 @@ pages:
   # API Changes: api/changes.md                 # page-tree = NEED_NEW_PAGE
 - Hooks:
   - Using hooks: hooks.md                        # page-tree = NEED_PAGE_MOVE to /hooks/usage.md
```

Comment:
@seanmadsen I wasn't aware of the discussion on the structure, after reading through it I think option 4 was the right choice. My concern was more that there's now no overview page like in the wiki. I think an autogenerated one which shows a list of the hooks by category would be nice, but it's something that could be something added on later if we're sure it's needed.